### ISSUE TITLE
Use Probot's decorated Octokit client instead of using Octokit directly

### ISSUE THREAD

**Issue by JamesMGreene** [github/learning-lab-components#51]
Related to #42, we should switch this library from using `@octokit/rest` as a direct dependency and instead be using the decorated Octokit client provided by `probot`.

Not only does it feature some beneficial enhancements but it is also more aligned with how GitHub Learning Lab itself actually uses Octokit.

**Comment by JamesMGreene** [comment]
To do this, we will first need to upgrade the version of `probot` being used in Learning Lab, plus #42 / #59 here, and then we can make sure of the newer Octokit `.endpoint()` methods [which describe the endpoint being used, implemented in `@octokit/rest >= 16.0.0`] from the decorated Octokit client provided by `probot` via the `context.github`/`context.user.client` parameters. 👌 

e.g. https://runkit.com/jamesmgreene/octokit-rest-validation-v15-vs-v16

```js
const octokit16 = require('@octokit/rest@16.28.6')()
console.log(octokit16.repos.get.endpoint())
```

![image](https://user-images.githubusercontent.com/417751/61835952-9fe2e600-ae43-11e9-823c-9d86b143fb2a.png)

**Comment by heiskr** [comment]
Closing per team discussion
